src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -340,6 +340,17 @@ public void jobLogic () {
             // Set baseUrl after success.
             status.baseUrl = otpServer.publicUrl;
         }
+
+        // Now that the main instance is updated successfully, update Pelias
+        if (deployment.peliasUpdate) {
+            // Get log upload URI from deploy job
+            AmazonS3URI logUploadS3URI = getS3FolderURI();
+
+            // Execute the pelias update job and keep track of it
+            PeliasUpdateJob peliasUpdateJob = new PeliasUpdateJob(owner, "Updating Custom Geocoder Database", deployment, logUploadS3URI);
+            JobUtils.heavyExecutor.execute(peliasUpdateJob);
```

Comment:
I was thinking we should use `addNextJob` like this:
```suggestion
            this.addNextJob(peliasUpdateJob);
```

Reply:
Didn't know that was possible thanks! Changed in 4b73157ca781655923e944dad2ed1ccb0c32ce05